Realm/ObjectStore/object_store.cpp

```diff
@@ -289,40 +315,16 @@ bool ObjectStore::create_tables(Group *group, ObjectStore::Schema &target_schema
                         break;
                     }
                     default:
-#ifdef REALM_ENABLE_NULL
-                        target_prop.table_column = table->add_column(DataType(target_prop.type), target_prop.name, target_prop.is_nullable);
-#else
-                        target_prop.table_column = table->add_column(DataType(target_prop.type), target_prop.name);
-#endif
+                        target_prop.table_column = table->add_column(DataType(target_prop.type),
+                                                                     target_prop.name,
+                                                                     target_prop.is_nullable);
                         break;
                 }
 
-                if (current_prop && property_can_be_migrated_to_nullable(*current_prop, target_prop)) {
```

Comment:
is this being removed intentionally? I thought we wanted the automatic migration?